ui/backend/client_ui_sqlite.py

```diff
@@ -165,3 +169,26 @@ def get_files_summary(self, repo_url):
                 " FROM discoveries WHERE repo_url=?"
                 " GROUP BY file_name"
             ))
+
+    def update_similar_snippets(self,
+                                target_snippet,
+                                state,
+                                repo_url,
+                                file_name=None,
+                                threshold=0.95):
```

Comment:
it was 0.96 in the parent class. Why have you changed it here?

Reply:
Indeed, I forgot to update the value here, I thought I had done so. It should be 0.96

Reply:
Oh but the code should have been removed from that class altogether, that's why I didn't update it. I'm removing it now